selfdrive/car/toyota/carcontroller.py

```diff
@@ -94,7 +95,11 @@ def update(self, CC, CS, now_nanos):
     can_sends.append(create_steer_command(self.packer, apply_steer, apply_steer_req))
     if self.frame % 2 == 0 and self.CP.carFingerprint in TSS2_CAR:
       lta_active = CC.latActive and self.CP.steerControlType == SteerControlType.angle
-      can_sends.append(create_lta_steer_command(self.packer, self.last_angle, lta_active, self.frame // 2))
+      need_to_winddown = False  # abs(CS.out.steeringAngleDeg) > abs(actuators.steeringAngleDeg) and abs(CS.out.steeringAngleDeg) - abs(actuators.steeringAngleDeg) > 4
+      enable_condition = abs(CS.out.steeringTorqueEps) < 1500 and abs(CS.out.steeringTorque) < 150 and not need_to_winddown
```

Comment:
need to understand how specifically this works. just varying this bit does not seem to have the same affect as looking at the EPS torque. this limits the torque (when user not touching wheel) perfectly to whatever you set it to

Reply:
Note: the enable bits completely drop torque immediately on the TSS 2.5 2023 RAV4s, not a viable way to limit torque. SETME_X64 at 0 drops torque slowly